pymemcached_test.py::test_noreply_pipeline fails

https://github.com/dragonflydb/dragonfly/actions/runs/11247452053/job/31270963227
```
        info = Redis(port=df_server.port).info()
>       assert info["total_pipelined_commands"] > len(keys) / 6  # sometimes CI is slow
E       AssertionError: assert 2 > (200 / 6)
E        +  where 200 = len(['k0', 'k1', 'k2', 'k3', 'k4', 'k5', ...])

```

--- a/tests/dragonfly/pymemcached_test.py
+++ b/tests/dragonfly/pymemcached_test.py
@@ -55,7 +55,7 @@ def test_noreply_pipeline(df_server: DflyInstance, memcached_client: MCClient):
     so all the commands are pipelined. Assert pipelines work correctly and the
     succeeding regular command receives a reply (it should join the pipeline as last).
     """
-    keys = [f"k{i}" for i in range(200)]
+    keys = [f"k{i}" for i in range(2000)]
     values = [f"d{i}" for i in range(len(keys))]
 
     for k, v in zip(keys, values):

